stemcell_builder/stages/rsyslog_config/assets/rsyslog_50-default.conf

```diff
@@ -10,35 +10,35 @@ auth,authpriv.*			/var/log/auth.log
 #cron.*				/var/log/cron.log
 daemon.*			-/var/log/daemon.log
 kern.*				-/var/log/kern.log
-lpr.*				-/var/log/lpr.log
-mail.*				-/var/log/mail.log
-user.*				-/var/log/user.log
+#lpr.*				-/var/log/lpr.log
+#mail.*				-/var/log/mail.log
+#user.*				-/var/log/user.log
```

Comment:
Could we delete these lines entirely from this file instead of leaving them commented?

Thanks,
Aakash & @jaresty

Reply:
I think we decided to keep them as this is how it looks like in the [default configuration for ubuntu](https://github.com/rsyslog/rsyslog-pkg-ubuntu/blob/99381a56fa1338d2d48c6e0aafd985226d8a7fb8/rsyslog/xenial/v8-stable/debian/50-default.conf#L9).

Reply:
Seems fair!